cypress/integration/e2e/sign-in-gate.spec.js

```diff
@@ -113,52 +160,333 @@ describe('Sign In Gate Tests', function() {
             cy.get('[data-cy=sign-in-gate-main]').should('not.be.visible');
         });
 
-        it('register button should contain profile.theguardian.com href', function() {
+        it('register button should contain profile.theguardian.com href', function () {
             visitArticleAndScrollToGateForLazyLoad();
 
             cy.get('[data-cy=sign-in-gate-main]').should('be.visible');
 
             cy.get('[data-cy=sign-in-gate-main_register]')
-                .invoke('attr', 'href').should('contains', 'profile.theguardian.com');
+                .invoke('attr', 'href')
+                .should('contains', 'profile.theguardian.com');
         });
 
-        it('sign in link should contain profile.theguardian.com href', function() {
+        it('sign in link should contain profile.theguardian.com href', function () {
             visitArticleAndScrollToGateForLazyLoad();
 
             cy.get('[data-cy=sign-in-gate-main]').should('be.visible');
 
-
             cy.get('[data-cy=sign-in-gate-main_signin]')
-                .invoke('attr', 'href').should('contains', 'profile.theguardian.com');
+                .invoke('attr', 'href')
+                .should('contains', 'profile.theguardian.com');
         });
 
-        it('should show cmp ui when privacy settings link is clicked', function() {
+        it('should show cmp ui when privacy settings link is clicked', function () {
             visitArticleAndScrollToGateForLazyLoad();
 
             cy.get('[data-cy=sign-in-gate-main]').should('be.visible');
 
             cy.get('[data-cy=sign-in-gate-main_privacy]').click();
+            getCMPiFrame().contains('privacy settings').should('be.visible');
```

Comment:
I think I may have fixed this in https://github.com/guardian/dotcom-rendering/pull/1829/files#diff-ae97a2c8603811d3bc137eddacb4a4f1L142 but I didn't seem to have to do the getCMPIframe bit so not sure why!